package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "diagram-js-ui",
+  "name": "@bpmn-io/diagram-js-ui",
```

Comment:
`package-lock.json` should be updated as well -> just run `npm i` so that it's done automatically